### ISSUE TITLE
Help with Promise Ring

### ISSUE THREAD

**Issue by Immortal911** [Monika-After-Story/MonikaModDev#3367]
I'm trying to give Monika the promise ring as a gift, but nothing seems to be working. I've made sure to spell it correctly and everything i.e. promisering.gift. I made sure to note that it's a gift file as well, but she can never read it. Is it only on special days she can actually recieve it or was it only a one-time only thing? I've tried giving her this for the past three days.

**Comment by multimokia** [comment]
How long have you been playing MAS for?

**Comment by Immortal911** [comment]
Ah, okay. Thank you for clearing that up for me. That's what I get for deleting that persistant data.

**Comment by multimokia** [comment]
Why did you clear it by the way?

**Comment by Immortal911** [comment]
I figured that I messed something up. I ended up missing alot of content from her birthday update, I figured that I wouldn't be able to get them back. I couldn't even get the hangman game to trigger either for some reason, but I didn't know that I just had to just play some of the games or just wait for her to trigger them herself. Looking back on it now, I'm facepalming myself for not coming here and asking about it. There's not really alot of information on Reddit or online in general about some of the issues that I had before. So, I figured that clearing it will fix it somehow.
